documentation/Installing-Clear-Containers-on-ClearLinux.md

```diff
@@ -1,4 +1,4 @@
-# Install the Clear Containers runtime on Clear Linux
+# How to install the Clear Containers runtime on Clear Linux
```

Comment:
FYI: This is a "hot" topic in technical writing. Infinitive vs. Gerund. At the moment, the industry has sided with the infinitive which TCS has also adopted. 'How to install' is considered the long form of 'Install'.  'Installing' is not acceptable only due to consistency.